vogue/commands/load/flowcell.py

```diff
@@ -10,9 +10,9 @@
 
 LOG = logging.getLogger(__name__)
 
-@click.command("flowcell", short_help = "load flowcell into db.")
-@click.option('-r', '--run-id', help = 'Lims process id for the run. Eg: 24-100451')
-@click.option('-a', '--all-runs', is_flag = True, help = 'Loads all lims flowcells ids')
+@click.command("flowcell", short_help = "Load flowcell into db.")
+@click.option('-r', '--run-id', help = 'Flowcell id. Eg: 190315_D00410_0873_BHWWKCBCX2')
```

Comment:
Same here: a flowcell id is HWWKCBCX2

Reply:
I changed back to lims process id witch is more correct in this case. We want to be able to load data from a specific process with this option.

Reply:
Can you then change the help field to reflect that? e.g. 
```suggestion
@click.option('-r', '--run-id', help = 'Run id. Eg: 190315_D00410_0873_BHWWKCBCX2')
```

Reply:
Im quite sure I did @ingkebil